dataeng/jobs/analytics/WarehouseTransformsCI.groovy

```diff
@@ -45,7 +45,7 @@ class WarehouseTransformsCI{
                 git {
                     remote {
                         url('$WAREHOUSE_TRANSFORMS_URL')
-                        refspec('+refs/pull/*:refs/remotes/origin/pr/*')
+                        refspec('+refs/heads/master:refs/remotes/origin/master +refs/pull/*:refs/remotes/origin/pr/*')
```

Comment:
origin/master branch was not being updated and with time it gets outdated in jenkins CI, it is important for master branch to be updated